[Form] Fixing "HTML5 color format" link

ktherage opened the pull request:
The link https://www.w3.org/TR/html52/sec-forms.html#color-state-typecolor redirects to https://html.spec.whatwg.org/multipage/forms.html#color-state-typecolor which is bad, it's seems to be https://html.spec.whatwg.org/multipage/input.html#color-state-(type=color) now.

ktherage committed:
Fixing "HTML5 color format" link

The link https://www.w3.org/TR/html52/sec-forms.html#color-state-typecolor redirects to https://html.spec.whatwg.org/multipage/forms.html#color-state-typecolor which is bad, it's seems to be https://html.spec.whatwg.org/multipage/input.html#color-state-(type=color) now.

javiereguiluz commented:
Thanks for this contribution! If possible, please check if this same error occurs in 5.4 branch, which is the oldest maintained branch that could contain the mistake, and try to rebase this pull request to that branch. Thanks!

ThomasLandauer committed:
Fixing path

All other code samples on this page are using `form/`

OskarStark committed:
minor #17264 [Form] Fixing path (ThomasLandauer)

This PR was merged into the 4.4 branch.

Discussion
----------

[Form] Fixing path

All other code samples on this page are using `form/`

Commits
-------

e6bdd5d51 Fixing path

javiereguiluz committed:
Merge branch '4.4' into 5.4

* 4.4:
  Fixing path

javiereguiluz commented:
Merged in `5.4`. Thanks!
